.travis.yml

```diff
@@ -10,17 +10,16 @@ compiler:
 - gcc
 - clang
 
-
-install:
+before_install:
```

Comment:
This should not be changed:

> before_install:
> execute all of the commands which need to be executed before installing dependencies

vs 

> install
> install all of the dependencies you need here

We are clearly installing dependencies here.